tabs/tab_baseline_matching.py

```
@@ -994,15 +1010,25 @@ def out_matched_stats():
             return render.DataGrid(d.groupby(treat)[var].describe().reset_index())
         return None
 
-    @render_widget
+    @render.ui
     def out_matched_boxplot():
```

Ruff (0.14.10)

Missing return type annotation for private function `out_matched_boxplot`

(ANN202)